docs/specifications/sad/SAD.md

```diff
@@ -87,7 +87,7 @@ hands.
 
 Architectural constraints for this project might be the interaction with widgets, since we do not allow the user to 
 upload executable code for security reasons. So the developer might be limited when it comes to the interaction of the 
-user with the developers widgets. But we do accept this limitation, because of the security goal. 
+user with the developers widgets. But we do accept this limitation, because of the security goal.
```

Comment:
Why did commit this change? Make sure that you don't commit these changes in the future.

Reply:
My fault, I didn't recognize that.